caluma/caluma_form/filters.py

```diff
@@ -325,6 +326,9 @@ class SearchAnswersFilter(Filter):
 
     def __init__(self, *args, **kwargs):
         self.document_id = kwargs.pop("document_id")
+        self.UNSUPPORTED_QUESTION_TYPES = [
+            type for type in Question.TYPE_CHOICES if type not in self.FIELD_MAP.keys()
+        ]
```

Comment:
This can be moved to class scope as a constant - no need to do it every time the filter is instantiated.

Better yet, remove the constant - it's the exclusion of FIELD_MAP, so we can just use that one to check for supported question types

Reply:
refactored away